usr/sbin/wakealarm

```diff
@@ -1,215 +1,154 @@
 #!/usr/bin/env python3
 """
-WakeAlarm script for OpenMediaVault - Sets RTC alarm based on schedule
+
+
+
 """
+from croniter import croniter
+from datetime import datetime
+from datetime import timedelta
+from sys import exit
+import time
+from time import mktime
 
-from datetime import datetime, timedelta
-import argparse
 import logging
 import logging.handlers
-import time
-from logging import NullHandler
-from pathlib import Path
-
-from croniter import croniter
 
 WAKEUP_CONF = '/etc/wakealarm.conf'
 WAKEALARM = '/sys/class/rtc/rtc0/wakealarm'
+#BOOTCOMPENSATION = timedelta(minutes=2)
+
+try:
+    from logging import NullHandler # Python >= 2.7
```

Comment:
I don't think Python < 3 has to be supported OMV is based on Python3. This is useless because the shebang clearly says Python3: `#!/usr/bin/env python3`